manuscript/08-Iterators-And-Generators.md

```diff
@@ -41,7 +41,7 @@ console.log(iterator.next());           // "{ value: undefined, done: true }"
 console.log(iterator.next());           // "{ value: undefined, done: true }"
 ```
 
-The `createIterator()` function in this example returns an object with a `next()` method. Each time the method is called, the next value in the `items` array is returned as `value`. When `i` is 4, `items[i++]` returns `undefined` and `done` is `true`, which fulfills the special last case for iterators in ECMAScript 6.
+The `createIterator()` function in this example returns an object with a `next()` method. Each time the method is called, the next value in the `items` array is returned as `value`. When `i` is 4, `value` returns `undefined` and `done` is `true`, which fulfills the special last case for iterators in ECMAScript 6.
```

Comment:
Value is a variable, it can't return anything. Items[I++] is an operation, so it can return something.